app/services/talents/search.rb

```diff
@@ -57,6 +57,8 @@ def filter_by_status(talents)
           .active
           .where("tokens.deployed_at > ?", 1.month.ago)
           .order("tokens.deployed_at ASC")
+      elsif filter_params[:status] == "Pending approval"
+        Talent.joins(:user).where(user: {profile_type: "waiting_for_approval"})
```

Comment:
we lose the previous filtering by doing this.

```suggestion
        talents.joins(:user).where(user: {profile_type: "waiting_for_approval"})
```

Reply:
Good catch. unfortunately your suggestion doesn't work because we do Talent.base before, which filters out the private talents. in this case we need both. I'll dm you to discuss the best solution